task.py

```diff
@@ -11,6 +11,23 @@
 class TaskState:
     PENDING, PROCESSING, COMPLETED, FAILED = range(4)
 
+    @staticmethod
+    def name(state):
+        """Return the name of the state"""
+        # There's almost certainly a better way to do this. On py3
+        # there's enum, but here we'll just inline it.
+        if state == TaskState.PENDING:
+            return 'PENDING'
+        elif state == TaskState.PROCESSING:
+            return 'PROCESSING'
+        elif state == TaskState.COMPLETED:
+            return 'COMPLETED'
+        elif state == TaskState.FAILED:
+            return 'FAILED'
```

Comment:
There's no `enum` in py2 or I would have used it :) (in the comment and commit message)